zcash_primitives/src/transaction/tests.rs

```diff
@@ -20,6 +16,12 @@ use super::{
     txid::TxIdDigester,
     Authorization, Transaction, TransactionData, TxDigests, TxIn,
 };
+use crate::transaction::OrchardBundle::OrchardVanilla;
+#[cfg(zcash_unstable = "nu6" /* TODO nu7 */ )]
+use crate::transaction::OrchardBundle::OrchardZSA;
+use crate::{
+    consensus::BranchId, legacy::Script, transaction::components::amount::NonNegativeAmount,
+};
```

Comment:
we should not move existing unrelated imports

Reply:
moved back